crates/sovereign-sdk/rollup-interface/src/state_machine/zk/mod.rs

```diff
@@ -37,7 +37,8 @@ pub trait ZkvmHost: Zkvm + Clone {
     /// The associated guest type
     type Guest: ZkvmGuest;
     /// Give the guest a piece of advice non-deterministically
-    fn add_hint<T: BorshSerialize>(&mut self, item: T);
+    /// `item` is a borsh serialized input to the guest
+    fn add_hint(&mut self, item: Vec<u8>);
```

Comment:
Why?

Reply:
had some problems regarding types idk

Reply:
We should fix it then?

Reply:
is it really important?

every adapter was doing borsh::to_vec()